src/components/FolderList.jsx

```diff
@@ -0,0 +1,22 @@
+const TOTAL_LIST_NAME = '전체';
+
+const FolderList = ({folderList, totalHandler, listHandler}) => {
+  if (!folderList.length) {
+    return <></>
+  }
```

Comment:
null로 리턴해주는게 명확한 것 같아요~
```suggestion
  if (!folderList.length) {
    return null;
  }
```

Reply:
(취향) 요런 느낌으로도 추천드립니다!
```suggestion
  if (folderList.length === 0) {
    return <></>
  }
```